### DISCUSSION TITLE
Trying to run a local debug build

### DISCUSSION THREAD

**Discussion by nazq** [starship/starship#6308] [Q&A] [answered]
Hi, I may be missing something but I have a local build from source and am trying to figure out how to run starship in the local terminal without affecting my global stable install. Can someone point me in the right direction. 

Thanks

**Comment by davidkna** [comment, marked as answer]
You can use the CLI to render the prompt via `cargo run -- prompt`, `target/debug/starship prompt`. Of course, you can also run tests via `cargo test`.
For testing changes related to the shell integration, you should reset `PATH` and use `--print-full-init` to source the script, e.g., `PATH= ./target/debug/starship init zsh --print-full-init` and `eval` the output to avoid starship picking up the wrong render binary.
